### PR TITLE
Bottom bar fixed app, info and added a clean `guide_manual.png`

### LINKED ISSUE #418: Bottom bar comes of the top, both in the app & `info` and `guide_manual.png` is messy.
**I'm submitting a ...**
- [x] bug report(s)
- [ ] feature request

**Current behavior:**
* Multiple bugs in bottom bar.
> It floats when `textfield` is manipulated.

badgeyay_issue

* When we click on `info`, the bar comes in between the screen and stays there.
image

---
* **Image is messy and ill-presented.**
image

**Expected behavior:**
* It should stick to the bottom :+1:  _(for 1 & 2)_
* The image should be re created. :smile: 

**Steps to reproduce:**
See the **GIF**. :wink: 

## WIP

### PR DESCRIPTION
**Pull request by 0x48piraj**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- **fixed** floating when `text field` was manipulated.
- **fixed** floating of bar in `info`
- **Re-created** a clean version of `guide_manual.png`
image

Signed-off-by: Piyush Raj

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@0x48piraj One PR is already there fixing one of the issue, why you opened new PR?
So [sagar-kalra](https://github.com/sagar-kalra) opened PR first so he should get fair chance so we will merge his PR if the PR is perfect.
Thanks

File: app/static/css/style.css [hunk 1]
```diff
@@ -197,7 +197,7 @@ textarea {
 }
 
 #footer {
-    position: absolute;
+    position: fixed;
```
Comment (harshit98) [inline app/static/css/style.css]: Remove this line. This fix has been already provided in other PR.

**Comment by 0x48piraj** [comment]
@vaibhavsingh97 
I claimed it first, so I should get that chance, also, It's written to ask the issue filer before claiming, he just put a PR in between.
And @yashLadha said to break the issues into pieces.
After that @harshit98 said, compile them to one as terms like **WIP** are not used.

File: app/static/css/style.css (hunk 1 above)
Comment (0x48piraj) [reply app/static/css/style.css]: I have said him to close that PR, as he broke the rules.

**Comment by vaibhavsingh97** [comment]
Oops!! Looks like lot of confusion here. No problem, we will take care

**Comment by harshit98** [comment]
Okay. I have merged the pull request of @sagar-kalra. This pull request provides only change in `guide_manual.png`. Squash your commits @0x48piraj. Then we're good to go!

**Comment by 0x48piraj** [comment]
**I was on web-interface.**
But, @harshit98 I read on **SO** that, the maintainer/owner can squash the commits easily :smile: 

> I will take care of that next time though.

image

##### Reference : https://stackoverflow.com/questions/35948131/github-squash-commits-from-web-interface-on-pull-request-after-review-comments

**Comment by gabru-md** [comment]
can always use git-cli to squash your commitsonce you're done editing. Editing does not necessarily has to be done from web gui or git cli. 
You can always clone back your repository and then rebase the head and squash yhe commits as far as i know.

**Comment by ParthS007** [comment]
@0x48piraj Please don't provide this excuse every time. You have made many PRs now and you should use git that's the basic thing you should know.

**Comment by 0x48piraj** [comment]
Closing this **PR** :+1: 

And, I'm sorry.
I will, from now on, squash every time, without seeing any **issue** requirements. 
From now on, All my PR's will only have just **1 commit** regardless of task length :heart:
It will never happen again.
I'm sorry for all the inconveniences I made.
I didn't meant anything, nor thought it will end up like this.
I'm so sorry @vaibhavsingh97 @harshit98 :cry:

**Pull request closed without merging**